Vungle/VungleAdapter/ALVungleMediationAdapter.m

```diff
@@ -140,7 +141,14 @@ - (void)destroy
     if ( self.adView )
     {
         // Note: Not calling this for now because it clears pre-loaded/cached ad view ads as well.
-        // [[VungleSDK sharedSDK] finishedDisplayingAd];
+        if (self.bidResponse) {
```

Comment:
Need to add spaces and move the brace to the other line to follow MAX standards

Reply:
Line